IdentityCore/src/cache/MSIDToken.m

```diff
@@ -33,17 +33,62 @@
 //in seconds, ensures catching of clock differences between the server and the device
 static uint64_t s_expirationBuffer = 300;
 
+@interface MSIDToken ()
+
+@property (readwrite) NSString *token;
```

Comment:
just curious, why all readwrite properties are necessary?

Reply:
You are right, we need only 'json' property here.